### PR TITLE
Update Staking Docs - Bags-list

### LINKED ISSUE #3156: Explain Bags on Polkadot JS UI
From Kusama Validator Lounge - 
> Hi. What does this new "Bags" selection do on polkadot.js? Anyone able to point me to the documentation?
image

### PR DESCRIPTION
**Pull request by DrW3RK**
Preliminary update of Staking docs with Bags-list updates

### CODE REVIEW

**Comment by DrW3RK** [comment]
@emostov @kianenigma Let us know if there are any gaps to be addressed regarding bags-list on the Wiki. Can make additional updates after this Apps UI PR gets merged https://github.com/polkadot-js/apps/pull/7179

File: docs/learn/learn-nominator.md
Comment (kianenigma) [inline docs/learn/learn-nominator.md]: This is only the case with the "current parameters". In principle bags-list can store an infinite number of nodes. We should emphasize: 

'in the current config, we keep 50k in bags list and 22,5k come out as the electing'

File: docs/learn/learn-nominator.md
Comment (kianenigma) [inline docs/learn/learn-nominator.md]:
```suggestion
of which, the stake of the top 22,500 nominators is considered for elections that eventually determine the active validators. The bags-list
```

File: docs/learn/learn-nominator.md
Comment (kianenigma) [inline docs/learn/learn-nominator.md]:
```suggestion
cause an account to be in the wrong bag, and for scalability reasons the system will not automatically
```

File: docs/learn/learn-nominator.md
Comment (kianenigma) [inline docs/learn/learn-nominator.md]: You talk about the last bag here, which is an important concept, but you have not explained what is, and why it matters. In other words, how the bags are used to iterate over all nominator intentions is missing. I would include this explanation + some graphics about it beforehand as well. Example of what I have in mind is the "bags-list in depth" here: https://docs.google.com/document/d/1RMoTpvXyoudMeVTRJ-a6p9XDpdCfUovRapJINZJB4hI/edit#

Comment (profradha) [reply docs/learn/learn-nominator.md]: Noted

File: docs/learn/learn-nominator.md
Comment (kianenigma) [inline docs/learn/learn-nominator.md]: _rewards_ is not really a relate term here. I think what you are trying to say is that "iteration" is based on insertion order, so when we want to _iterate_ the bags-list to go from nomination intentions to the electing subset, then the insertion order matters within a bag.

File: docs/learn/learn-nominator.md
Comment (kianenigma) [reply docs/learn/learn-nominator.md]: Okay I see some explanation along those lines further below, but I think it is better to make it more comprehensive and move it to the top.

File: docs/learn/learn-nominator.md
Comment (kianenigma) [inline docs/learn/learn-nominator.md]: An example of rebag will also be great, as noted above.

File: docs/learn/learn-staking.md
Comment (kianenigma) [inline docs/learn/learn-staking.md]: this is a great idea! we should 100% make 22500 and 50,000 and 10 DOT a also variables.

Comment (kianenigma) [reply docs/learn/learn-staking.md]: If you want to be really fancy and web3, you can also and actually should read them from the chain state or metadata upon creation of these web pages.

File: docs/learn/learn-staking.md
Comment (kianenigma) [inline docs/learn/learn-staking.md]: do you explain anywhere the difference between intention, electing, and active? then it might be a good idea to put some footnotes about it here and there.

Comment (profradha) [reply docs/learn/learn-staking.md]: Yeah. Can add a reference to it https://wiki.polkadot.network/docs/learn-nominator#staking-election-stages

File: docs/learn/learn-staking.md
Comment (profradha) [reply docs/learn/learn-staking.md]: Indeed. I wanted the content reviewed first. Will update the numbers to variables which can actually fetch from the chain state https://github.com/w3f/polkadot-wiki/blob/master/scripts/inject-dict.json

**Comment by DrW3RK** [comment]
@kianenigma Your feedback has been addressed. Please review and approve if everything looks okay

File: docs/learn/learn-nominator.md
Comment (kianenigma) [inline docs/learn/learn-nominator.md]: should we use the term "electing" and reference to where it is explained?

File: docs/learn/learn-nominator.md
Comment (kianenigma) [inline docs/learn/learn-nominator.md]: misleading: bags are ordered and are iterated from high to low, WITHIN a bag we use insertion order.

File: docs/learn/learn-nominator.md
Comment (kianenigma) [inline docs/learn/learn-nominator.md]: Don't we now have variables for `50k` and `22,5k`?

File: scripts/inject-dict.json
Comment (kianenigma) [inline scripts/inject-dict.json:86]:
```suggestion
  {
    "tpl": "max_nominator_count",
    "default": 50000,
    "path": "query.staking.maxNominatorsCount"
  },
  {
    "tpl": "max_active_nominator_count",
    "default": 22500,
    "path": "consts.electionProviderMultiPhase.maxElectingVoters"
  },
```

https://polkadot.js.org/docs/substrate/constants#maxelectingvoters-u32

**Review by kianenigma — APPROVED** [review]
LGTM other than minor things I noted.

File: docs/learn/learn-nominator.md
Comment (profradha) [reply docs/learn/learn-nominator.md]: Will clarify

File: docs/learn/learn-nominator.md
Comment (profradha) [reply docs/learn/learn-nominator.md]: Will do

**Comment by DrW3RK** [comment]
@emresurmeli Please do one more pass and merge the PR

**Pull request merged**
